packages/core/src/client.ts

```diff
@@ -33,11 +33,16 @@ export interface WithAccountId extends Storage {
   accountId: () => Promise<Ref<Account>>
 }
 
+/**
+ * @public
+ */
+export interface WithFiles extends WithAccountId, FileStorage {}
+
 /**
  * Client with hierarchy and model inside. Allow fast search for model, without accesing server.
  * @public
  */
-export interface Client extends WithAccountId {
+export interface Client extends WithFiles {
```

Comment:
I think we should rename `WithAccountId` to `CoreClient` and not making new one WithFiles.